src/index.ts

```diff
@@ -26,5 +27,13 @@ extendEnvironment((hre) => {
       return deploy(hre, contractName, ...args);
     }
   );
+    hre.deployScillaWithAccount = lazyFunction(
```

Comment:
Do we want this if we have connect() ?

Reply:
Probably not but I'll leave it to a future commit to remove this in case it needs more thought.